driver/src/main/java/org/neo4j/driver/v1/Config.java

```diff
@@ -67,6 +72,7 @@ private Config( ConfigBuilder builder )
 
         this.encryptionLevel = builder.encruptionLevel;
         this.trustStrategy = builder.trustStrategy;
+        this.authToken = builder.authToken;
```

Comment:
Isn't  connecting to a URL with AuthToken *the* most common thing todo with a driver? Why does a user have to care about setting up a config to achieve that goal?